harness/tests/src/main/kotlin/godot/tests/Entry.kt

```diff
@@ -1,21 +1,121 @@
 package godot.tests
 
-import godot.core.KtConstructor0
-import godot.core.KtFunction1
-import godot.core.KtVariant
+import godot.core.*
 
 class Entry : godot.runtime.Entry() {
-    override fun Context.init() {
-        with(registry) {
-            registerClass<Invocation>(Invocation::class.qualifiedName!!, "Spatial") {
-                constructor(KtConstructor0(::Invocation))
-                function(KtFunction1("int", Invocation::int, ::KtVariant, KtVariant::asInt))
-                function(KtFunction1("long", Invocation::long, ::KtVariant, KtVariant::asLong))
-                function(KtFunction1("float", Invocation::float, ::KtVariant, KtVariant::asFloat))
-                function(KtFunction1("double", Invocation::double, ::KtVariant, KtVariant::asDouble))
-                function(KtFunction1("boolean", Invocation::boolean, ::KtVariant, KtVariant::asBoolean))
-                function(KtFunction1("string", Invocation::string, ::KtVariant, KtVariant::asString))
-            }
-        }
-    }
-}
\ No newline at end of file
+	override fun Context.init() {
+		with(registry) {
+			registerClass<Invocation>(Invocation::class.qualifiedName!!, "Spatial") {
+				constructor(KtConstructor0(::Invocation))
```

Comment:
spaces instead of tabs :P Most likely IJ is picking up the `.editorconfig` of `godot`. You can turn that of in the settings

Reply:
Did not catch this, thx